packages/components/src/DateTimePickers/InputDateTimePicker/date-extraction.js

```diff
@@ -76,14 +84,17 @@ function dateTimeToStr(date, time) {
 		return format(date, INPUT_DATE_ONLY_FORMAT);
 	}
 
-	const { hours, minutes } = time;
+	const { hours, minutes, seconds } = time;
 	try {
-		const timeInMinutes = hoursAndMinutesToTime(hours, minutes);
-		const fullDate = setMinutes(date, timeInMinutes);
-		return format(fullDate, INPUT_FULL_FORMAT);
+		const timeInSeconds = hoursAndMinutesAndSecondsToTime(hours, minutes, seconds);
+		const fullDate = setSeconds(date, timeInSeconds);
+		return format(fullDate, seconds === '00' ? INPUT_WITHOUT_SECOND_FORMAT : INPUT_FULL_FORMAT);
```

Comment:
I don't think we should manage the 2 formats. It will be very hard to manage.
Just use the format with seconds, we should tackle the format change (by user or language).

Reply:
useSeconds to make the switch between format

dateTimeToStr(date, time, option) {useTime, useSecond} // use time in another PR